app/models/tokite/hook_event/pull_request_review_comment.rb

```diff
@@ -5,7 +5,7 @@ def fields
         {
           event: "pull_request_review_comment",
           repo: hook_params[:repository][:full_name],
-          body: hook_params[:comment][:body],
+          body: hook_params[:comment][:body] || "",
```

Comment:
Does pull_request_review_comment event without the comment body really exist...?

Reply:
How about when attaching files without comment ? (I still do not check this case...)

Reply:
I understand this case.

Reply:
I don't know whether the event really occur with empty comment body. So should these code be removed?